ts/features/bonus/siciliaVola/store/actions/voucherGeneration.ts

```diff
@@ -35,9 +53,122 @@ export const svGenerateVoucherFailure = createStandardAction(
   "SV_GENERATE_VOUCHER_FAILURE"
 )<string>();
 
+/**
+ * First step of the voucher generation: category selection
+ */
+export const svGenerateVoucherSelectCategory = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_CATEGORY"
+)<SvBeneficiaryCategory>();
+
+/**
+ * Step for worker or sick, to confirm that the annual income is under the threshold limit
+ */
+export const svGenerateVoucherUnderThresholdIncome = createStandardAction(
+  "SV_GENERATE_VOUCHER_UNDER_THRESHOLD_INCOME"
+)<boolean>();
+
+/**
+ * Step for worker, to indicate his company data
+ */
+export const svGenerateVoucherSelectCompany = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_COMPANY"
+)<Company>();
+
+/**
+ * Step for sick, to indicate the hospital data
+ */
+export const svGenerateVoucherSelectHospital = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_HOSPITAL"
+)<Hospital>();
+
+/**
+ * Step for student, to indicate his university data
+ */
+export const svGenerateVoucherSelectUniversity = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_UNIVERSITY"
+)<University>();
+
+export type FlightsDate = {
+  departureDate: Date;
+  returnDate?: Date;
+};
+
+/**
+ * Step for student, worker and sick to indicate the departure and the return date
+ */
+export const svGenerateVoucherSelectFlightsDate = createStandardAction(
+  "SV_GENERATE_VOUCHER_SELECT_FLIGHTS_DATE"
+)<FlightsDate>();
+
+/**
+ * get and handle available destination for a voucher request
+ */
```

Comment:
Could you add a TODO just to remember and clarify that the `request` type will be defined in a next iteration?

Reply:
Added in [1aa3371](https://github.com/pagopa/io-app/pull/3170/commits/1aa33711c50bbf9de121bfe0e55cd0298a088433)